frontend/src/__tests__/cypress/cypress/utils/testConfig.ts

```diff
@@ -51,12 +51,15 @@ const AWS_PIPELINES: AWSS3Buckets = {
     testConfig?.S3.AWS_SECRET_ACCESS_KEY || env.AWS_PIPELINES_BUCKET_SECRET_ACCESS_KEY || '',
   BUCKET_2: AWS_PIPELINES_BUCKET_DETAILS,
 };
+//export const BASE_URL = testConfig?.ODH_DASHBOARD_URL || env.BASE_URL || '';
+const TEST_NAMESPACE = testConfig?.APPLICATIONS_NAMESPACE;
```

Comment:
Maybe it's better to have some kind of default deffinition like:
`const TEST_NAMESPACE = testConfig?.APPLICATIONS_NAMESPACE ||  'rhods-application';`